src/analyses/base.ml

```diff
@@ -163,6 +163,11 @@ struct
     | Le -> ID.le
     | Ge -> ID.ge
     | Eq -> ID.eq
+    (* TODO: This causes inconsistent results:
+       def_exc and interval definitely in conflict:
+         evalint: base eval_rv m -> (Not {0, 1}([-31,31]),[1,1])
```

Comment:
I'm not sure I understand this. `(Not {0, 1}([-31,31]),[1,1])` already is an invalid value in the first place!

Reply:
Yes, but `IntDomTuple` doesn't know that if int domain refinement is not applied: it's an unreduced product, so we don't consider such code dead immediately but possibly only at some later operation. Tracing the SV-COMP program shows how such inconsistent values in the TODOs come about during actual analysis. I think the first inconsistent value there arises from intervals not being able to express the exclusion set of 0 (which comes from a non-equality with 0) for refinement, so it's a top interval instead, which doesn't refine the interval but def_exc can be refined.